### ISSUE TITLE
Lighthouse

### ISSUE THREAD

**Issue by DinneK** [DrSLMac/pantry-to-pan#86]
- [ ] You must score as close to 100% as possible with the “Lighthouse Accessibility Audit”. 
- [ ] Be prepared to explain any accessibility audits your application is failing.
